### PR TITLE
Lesson-1-3

### PR DESCRIPTION
**Pull request by diorshelton**
Lesson 1.3 assignment

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -1,11 +1,22 @@
 import React from "react";
 
-const AddToDoForm = () => {
+const AddToDoForm = (props) => {
+
+	const handleAddToDo = (event) => {
+		event.preventDefault();
+		let todoTitle = event.target.title.value;
+		console.log(todoTitle);
```
Comment (dorianmariecom) [inline src/AddTodoForm.js:8]: you left a logging statement, maybe you want to use eslint to avoid that (it's a linter)

File: src/TodoListItem.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+const TodoListItem = (props) => {
+  return (
+    <li> {props.item.title}</li>
```
Comment (dorianmariecom) [inline src/TodoListItem.js:5]: You have extra unecessary spacing

File: src/TodoListItem.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  )
+}
+
+export default TodoListItem;
```
Comment (dorianmariecom) [inline src/TodoListItem.js:9]: You should configure your editor or use an `.editorconfig` file to have consistent newlines at the end of files

File: src/AddTodoForm.js (hunk 1 above)
Comment (diorshelton) [reply src/AddTodoForm.js:8]: Thanks for the suggestion. Do you have any thoughts on which style guide I should use?

**Comment by dorianmariecom** [comment]
I have more opinions about Ruby, in js I like to let petite do the
formatting and any eslint preset is fine

File: src/TodoListItem.js (hunk 3 above)
Comment (diorshelton) [reply src/TodoListItem.js:9]: Is this a plug-in? Does it format like Prettier and ESLint?

**Comment by dorianmariecom** [comment]
Sorry autocorrect I meant prettier

**Pull request merged**